### ISSUE TITLE
Disable Automatic Update

### ISSUE THREAD

**Issue by BhupeshManimaran** [adoptium/installer#1029]
## Need to Disable the Automatic Update for the installed JDK and JRE.

We have installed JAVA JDK and JRE which gets Updated automatically updated without showing like there is an update available. We have also checked for the availability of JAVA Updater Service or the Application in the Notification Tray Icon of Windows 11, we did not find any.
Request you to support us on disabling the JDK and JRE.

## Context

Java version: 
JDK - OpenJDK11U-jdk_x64_windows_hotspot_11.0.17_8
JDK - OpenJDK11U-jre_x64_windows_hotspot_11.0.17_8

Your operating system and platform:
Windows 11 Pro 64bit

**Comment by karianna** [comment]
@BhupeshManimaran 

Whilst we look into this, can you list the properties of the original installer you used? Here’s how to do it:

Run the MSI with logging enabled:

`msiexec /i <TheOriginalInstaller>.msi /l*v log.txt`

Open the generated `log.txt` file and search for the line starting with `PROPERTY`. This will display the properties used during the installation process.  See if there's any AUTO or UPDATE type of property listed there.

**Comment by BhupeshManimaran** [comment]
@karianna

Yes, I have generated Log and was able to find the below Property while generating logs for both JDK - OpenJDK11U-**jdk_x64_windows_hotspot_11.0.17_8.msi** & JRE - **OpenJDK11U-jre_x64_windows_hotspot_11.0.17_8.msi**:
Property(C): ARPURLUPDATEINFO = https://adoptium.net/releases.html

**Comment by karianna** [comment]
@jmjaffe37 - Do you know if we set a property for auto updating (true or false) with the new Wix 5 installers?

**Comment by jmjaffe37** [comment]
I just looked through the codebase again and I was not able to find any auto updating flags set within. I also looked through the [wixtoolset documentation](https://wixtoolset.org/docs/fivefour/) (we currently use version 5.0.0, but will soon move to 5.0.2), and I could not find settings/flags for auto-updating either.

Feel free to take another look incase I missed anything

**Comment by BhupeshManimaran** [comment]
@karianna & @jmjaffe37 So there is no possibility for the Adoptium JDK/JRE to Auto Update right?
Let me connect with you if I face this again. So that we can understand better.
